frontend/talentsearch/src/js/components/roleSalaryForm/RoleSalaryForm.tsx

```diff
@@ -30,6 +29,26 @@ export interface RoleSalaryFormProps {
   onUpdateRoleSalary?: RoleSalaryUpdateHandler;
 }
 
+// accessible button for modals - generate clickable inline elements resembling <a>
+interface ModalButtonProps {
+  click: (e: React.MouseEvent<HTMLSpanElement, MouseEvent>) => void;
+  children?: React.ReactNode;
+}
+const ModalButton: React.FC<ModalButtonProps> = ({ click, children }) => {
+  return (
+    <Button
+      color="black"
+      mode="inline"
+      data-h2-padding="b(all, none)"
+      data-h2-font-weight="b(200)"
```

Comment:
Do you really want a lighter font?  In the wireframe it looks about the same.  200 is pretty light (though you can't see it in Chrome on Windows).
![image](https://user-images.githubusercontent.com/8978655/169572153-5f2c916f-0308-4967-8056-aa7af80a2bfa.png)